custom_connectors/oauth2/bim_360.rb

```diff
@@ -2814,35 +1910,48 @@
       ],
       input_fields: lambda do |_object_definitions|
         [
-          { name: 'file_name', optional: false,
+          { name: 'file_name', optional: false, label: 'File Name',
```

Comment:
Update label as
```
label: 'File Name',
```
You can ignore as well, workato derives labels from name field.